botocore/handlers.py

```diff
@@ -581,6 +581,27 @@ def document_glacier_tree_hash_checksum():
     return AppendParamDocumentation('checksum', doc).append_documentation
 
 
+def document_cloudformation_get_template_return_type(section, event_name, **kwargs):
+    if 'response-params' in event_name:
+        template_body_section = section.get_section('TemplateBody')
+        type_section = template_body_section.get_section('param-type')
+        type_section.clear_text()
+        type_section.write(':type TemplateBody: dict')
+    elif 'response-example' in event_name:
+        parent = section.get_section('structure-value')
+        param_line = parent.get_section('TemplateBody')
+        value_portion = param_line.get_section('member-value')
+        value_portion.clear_text()
+        value_portion.write('{"AWSTemplateFormatVersion" : "version date", '
```

Comment:
Lets leave this as an empty dict. Since this is modeled as a string the structure could get updated without us knowing. There's a link in the description to the actual structure so it should be okay.